L1Trigger/Configuration/python/L1Trigger_EventContent_cff.py

```diff
@@ -117,7 +117,10 @@ def _appendStage2Digis(obj):
 # adding HGCal L1 trigger digis
 def _appendHGCalDigis(obj):
     l1HGCalDigis = [
-        'keep *_hgcalTriggerPrimitiveDigiProducer__*',
+        'keep *_hgcalTriggerPrimitiveDigiProducer_calibratedTriggerCells_*',
```

Comment:
@jbsauvan 
What is the reason for further specialization of `hgcalTriggerPrimitiveDigiProducer`  data too keep in these lines?  What are we dropping?

Reply:
We are dropping:
- The collection of `DataFormats/L1THGCal/HGCFETriggerDigi`, which are basically vectors of bool encoding the trigger cell raw charge. `HGCalTriggerCell` objects contain more information (e.g. calibrated energies) and are easier to use.
- The collection of `DataFormats/L1THGCal/HGCalTowerMap`, which are objects used to compute the towers

Reply:
*  >The collection of DataFormats/L1THGCal/HGCFETriggerDigi, which are basically vectors of bool encoding the trigger cell raw charge. HGCalTriggerCell objects contain more information (e.g. calibrated energies) and are easier to use.

OK, there we don't lose any info, I understand. 

 * > The collection of DataFormats/L1THGCal/HGCalTowerMap, which are objects used to compute the towers

Here there might be some info lost that we might need in the future.  Could not tell for sure
what L1T algo development could need. 

I rather would not drop anything from HGC digis, unless completely necessary.

Reply:
The HGCAL TPGs are only the 3D clusters and the towers. All other objects will not be available to the L1T system.
But ok, I can add the `HGCalTowerMap` objects, which give the tower energies in each layer.

Reply:
Remember to propagate the update to #22387 as well

Reply:
Thanks for the reminder; Yes I will.